web_widget_digital_signature/static/src/js/digital_sign.js

```diff
@@ -0,0 +1,131 @@
+openerp.web_widget_digital_signature = function(instance) {
+    var _t = instance.web._t;
+    var QWeb = instance.web.qweb;
+    var images = {}
+    instance.web.form.widgets.add('signature', 'instance.web.form.FieldSignature');
```

Comment:
all through this file, you sometime declare your variables and sometimes not. consider using `"use strict";`